quiche/src/h3/mod.rs

```diff
@@ -603,6 +614,98 @@ pub enum Event {
     GoAway,
 }
 
+/// Extensible Priorities parameters.
```

Comment:
Maybe expand this a bit to say that the priority object can be generated by parsing an extensible priority value with the `TryFrom` thing?

Reply:
PTAL